ovs-service/src/main/java/org/dcsa/ovs/util/pagination/Cursor.java

```diff
@@ -0,0 +1,48 @@
+package org.dcsa.ovs.util.pagination;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Value;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+/**
+ * A cursor implementation that maps to how spring data does paging.
+ */
+@Value
+public class Cursor {
+  private final int page;
+  private final int pageSize;
+  private final Sort.Direction direction;
+  private final String[] sortFields;
```

Comment:
In DCSA, every field can have a different direction.  Like

`sort=foo:ASC,Bar:DESC`

This model does not seem to capture this?

Reply:
Done